external-crates/move/crates/move-analyzer/src/bin/move-analyzer.rs

```diff
@@ -25,6 +25,10 @@ use move_analyzer::{
 use url::Url;
 use vfs::{impls::memory::MemoryFS, VfsPath};
 
+const LINT_NONE: &str = "none";
+const LINT_DEFAULT: &str = "default";
+const LINT_ALL: &str = "all";
```

Comment:
nit given these are three mutually exclusive options driving control flow it seems like a good case for using an `enum` and `match`ing on it

Reply:
but I suppose this way the constants correspond to the TS definitions without a conversion, so not a big deal

Reply:
Yeah, I wasn't sure what's best and kept in-sync with TS side of things